src/components/Overview/Overview.react.js

```diff
@@ -54,9 +87,22 @@ class Overview extends Component {
       'SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots.';
     //  Scrolling to top of page when component loads
     scrollToTopAnimation();
+    setInterval(() => {
```

Comment:
You also need to remove the interval on component un-mounting to prevent memory leaks.
Use
```
componentDidMount() {
    this.exampleTimer = setInterval ( () => {...} )
}

componentWillUnmount() {
    clearInterval(this.exampleTimer)   
}
```